### DISCUSSION TITLE
Brakująca funkcjonalność w /order/refund-claims

### DISCUSSION THREAD

**Discussion by javaguru-pl** [allegro/allegro-api#4526] [Suggestions - rejected]
:bulb: **Sugestia**
Wydaje mi się iż w parametrach metody **/order/refund-claims** brakuje parametru typu **timestamp**: **lastUpdateNewerThan**, który pozwalał by pobrać TYLKO te "claim-y" których **status** uległ zmianie PO dacie **lastUpdateNewerThan**.
Dzięki temu można by szybko i sprawnie pobierać tylko te claim-y, które zostały zaktualizowane od momentu ostatniego pobrania claim-ów (zamiast pobierać każdorazowo wszystkie i sprawdzać które zmieniły status, a które nie)

Taka zmiana pozwoli znacznie ograniczyć ilość requestów wysyłanych do tej metody API.
**Uda się Wam to wdrożyć w miarę szybko???**

Z góry dzięki za pomoc.
Mateusz

**Comment by Lukasz-Zurek** [comment]
Dziękuję za sugestię. Zgodnie z [naszym newsem](https://github.com/allegro/allegro-api/issues/4337) przenoszę wątek do zakładki [Discussions](https://github.com/allegro/allegro-api/discussions). Nie jestem jednak w stanie zadeklarować czy zmiana zostanie wdrożona i ewentualnego terminu jeśli to nastąpi.

**Comment by PrzemyslawLukanowski** [comment]
Dziękujemy za sugestię i opisanie potrzeby. Zaktualizowaliśmy jej status na „odrzucona”. Na ten moment nie planujemy dodawać opisanego parametru.
